tests/datasets_tests/test_zinc.py

```diff
@@ -0,0 +1,113 @@
+import os
+
+import numpy
+import pytest
+
+from chainer_chemistry.dataset.preprocessors.atomic_number_preprocessor import AtomicNumberPreprocessor  # NOQA
+from chainer_chemistry.datasets import zinc
+
+
+ZINC250K_NUM_LABEL = 3
+ZINC250K_NUM_DATASET = 249455
+
+
+def test_get_zinc_filepath_without_download():
+    filepath = zinc.get_zinc250k_filepath(download_if_not_exist=False)
+    if os.path.exists(filepath):
+        os.remove(filepath)  # ensure a cache file does not exist.
+
+    filepath = zinc.get_zinc250k_filepath(download_if_not_exist=False)
+    assert isinstance(filepath, str)
+    assert not os.path.exists(filepath)
+
+
+def test_get_zinc_filepath_with_download():
+    filepath = zinc.get_zinc250k_filepath(download_if_not_exist=False)
+    if os.path.exists(filepath):
+        os.remove(filepath)  # ensure a cache file does not exist.
+
+    # This method downloads the file if not exist
+    filepath = zinc.get_zinc250k_filepath(download_if_not_exist=True)
+    assert isinstance(filepath, str)
+    assert os.path.exists(filepath)
+
+
+@pytest.mark.slow
+def test_get_zinc():
+    # test default behavior
+    pp = AtomicNumberPreprocessor()
+    dataset = zinc.get_zinc250k(preprocessor=pp)
+
+    # --- Test dataset is correctly obtained ---
+    index = numpy.random.choice(len(dataset), None)
+    atoms, label = dataset[index]
+
+    assert atoms.ndim == 1  # (atom, )
+    assert atoms.dtype == numpy.int32
+    assert label.ndim == 1
+    assert label.shape[0] == ZINC250K_NUM_LABEL
+    assert label.dtype == numpy.float32
+
+    # --- Test number of dataset ---
+    assert len(dataset) == ZINC250K_NUM_DATASET
+
+
+def test_get_zinc_smiles():
```

Comment:
Please add `@pytest.mark.slow`

Reply:
This is intentional. I used `target_index` so this test is not slow.